### ISSUE TITLE
Quartermaster install adds sailfish-mdm

### ISSUE THREAD

**Issue by piggz** [black-sheep-dev/harbour-quartermaster#32]
Due to the use of mdm-battery info, sailfish-mdm is added when the app is installed, and prompts quite a scary warning about the device being managed.  Perhaps another method could be used to get battery status?

**Comment by black-sheep-dev** [comment]
Yes I used sailfish-mdm to get a unique device id for registration to the homeassistant server and also the battery status. It was a bad decision in the past, but it seemed to be a convenient way to  get these info.
For battery status i already have an other solution and also for a unique device id. A new version is in progress without mdm dependencies, but not finished yet.

**Comment by Thaodan** [comment]
You can use the machine-id to get the device-id. Check SSU for reference.
